code/parse/sexp/sexp_lookup.cpp

```diff
@@ -28,6 +28,7 @@ struct global_state {
 	int next_free_operator_id = First_available_operator_id;
 	int next_free_category_id = First_available_category_id;
 	int next_free_subcategory_id = First_available_subcategory_id;
+	int next_free_list_id = First_available_list_id;
```

Comment:
Similarly here, I'd prefer "enum_list" over "list" here, to help clue in people unfamiliar.